ebos/eclproblem.hh

```diff
@@ -914,7 +914,7 @@ public:
         // to the first "real" episode/report step
         // for restart the episode index and start is already set
         if (!initconfig.restartRequested()) {
-            simulator.startNextEpisode(schedule.seconds(0));
+            simulator.startNextEpisode(schedule.seconds(1));
```

Comment:
Why do you need this particular change?  Does the other simulator not run properly with a zero-sized step length passed into [`startNextEpisode()`](https://github.com/OPM/opm-models/blob/f1601ed1a833ef2f39109c96b1ac78473f08b4df/opm/models/utils/simulator.hh#L543-L548)?

Reply:
Else dt=0 at start of simulator.